docs/stylesheets/extra.css

```diff
@@ -1,40 +1,262 @@
-/* https://colorbrewer2.org/#type=qualitative&scheme=Paired&n=12 */
+/* Brand variables */
 :root {
-    --mri: #a6cee3;
-    --pet: #1f78b4;
-    --micr: #33a02c;
-    --meeg: #6a3d9a;
-    --beh: #fb9a99;
-    --nirs: #b2df8a;
-    --motion: #e31a1c;
-    --mrs: #fdbf6f;
-    --unused2: #ff7f00;
-    --unused3: #cab2d6;
-    --unused4: #ffff99;
-    --unused5: #b15928;
-    --key: #e41a1c;
-    --label: #377eb8;
-    --suffix: #4daf4a;
-    --ext: #984ea3;
-    --underscore: #ff7f00;
```

Comment:
we need to keep those otherwise we lose the colors on in several places in the getting starting guide:

in this PR:

<img width="874" height="565" alt="image" src="https://github.com/user-attachments/assets/eb3990d7-3c55-4861-8247-729ef17799d4" />


on main:

<img width="862" height="557" alt="image" src="https://github.com/user-attachments/assets/6080b716-2549-4ef5-ac41-75ebb82a4e48" />